tdms/src/hdf5_io.cpp

```diff
@@ -51,6 +53,7 @@ void HDF5Base::ls() const {
 }
 
 std::vector<hsize_t> HDF5Base::shape_of(const std::string &dataname) const {
+  SPDLOG_DEBUG("shape_of");
```

Comment:
I feel like we've touched on this before, but do we have a consistent style between the `SPDLOG_{macro}`s and `spdlog::{functions}`? 

Not a major thing, but it's probably good to stick to one or the other. And at a glance, the source code uses the functions so perhaps we go with them?

Reply:
Ah yes!

I like functions in the source, and macros are fine (and arguably more helpful) in the tests.